autosubmit/platforms/headers/pbs_header.py

```diff
@@ -0,0 +1,374 @@
+#!/usr/bin/env python3
+
+# Copyright 2017-2020 Earth Sciences Department, BSC-CNS
+
+# This file is part of Autosubmit.
+
+# Autosubmit is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+
+# Autosubmit is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+
+# You should have received a copy of the GNU General Public License
+# along with Autosubmit.  If not, see <http://www.gnu.org/licenses/>.
+
+import textwrap
+
+
+class PbsHeader(object):
+    """Class to handle the Lindgren headers of a job"""
+
+    # noinspection PyMethodMayBeStatic,PyUnusedLocal
+    def get_queue_directive(self, job, parameters, het=-1):
+        """
+        Returns queue directive for the specified job
```

Comment:
Would be good if we aligned on how pydoc comments are structured. This line you used is the third line of the documentation.

```
""" < --- first lline
 < --- empty, where sphinx/other tools would just add a break line
Returns queue... < -- your text
```

Some tools are able to handle this. But it's easier and more consistent if we follow something like https://peps.python.org/pep-0257/#multi-line-docstrings

We have both formats in our code. Old libs normally follow PEP-257. I noticed IDE/AI sometimes generate one or the other.